mod.ts

```diff
@@ -4,6 +4,12 @@ const SHA1_REGEX: RegExp = /^\s*sha-?1\s*$/i;
 const SHA256_REGEX: RegExp = /^\s*sha-?256\s*$/i;
 const SHA512_REGEX: RegExp = /^\s*sha-?512\s*$/i;
 
+export enum HashType {
```

Comment:
can u rename this to just `Hash`?
and make it a `const enum`?

Reply:
ah and please add a doc comment like all the other `export`s have

Reply:
Ok!

Reply:
> can u rename this to just `Hash`?
> and make it a `const enum`?

I cannot rename it to `Hash` because you are using already this name in a interface.
`export interface Hash `

Reply:
oh, overlooked that
can u just change that `export interface Hash` to `interface _Hash` - it actually shouldnt have been exported in the first place - thus the name doesnt matter much

then we could use that freed var name `Hash` for the enum